wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/ExpressionElement.scala

```diff
@@ -164,6 +164,12 @@ object ExpressionElement {
     override def arg1: ExpressionElement = prefix
     override def arg2: ExpressionElement = array
   }
+
+  final case class Suffix(suffix: ExpressionElement, array: ExpressionElement) extends TwoParamFunctionCallElement {
+    override def arg1: ExpressionElement = suffix
+    override def arg2: ExpressionElement = array
+  }
```

Comment:
`draft-3` is not supposed to support `suffix`, right?

Reply:
No, but all the other post-1.0 engine functions (ex. `min`, `max`, `sep`) have their case classes in here so I followed that convention. There aren't equivalent files in the other language version packages, though I can make that change if desirable.

Reply:
I feel like those functions may have been put in there before a next language version was declared and/or before we had our current organization scheme. I do think that separating them out (both current and the new addition) would be optimal, though maybe that's a separate ticket.

Reply:
Discussed off-github and decided to leave this as-is.